### ISSUE TITLE
Many disabled ("xtest_") tests in test_selenium.py - are they even needed?

### ISSUE THREAD

**Issue by jseppi** [18F/calc#1661]
I just noticed that in the old (from CALC 1 days) `test_selenium.py` tests, there are a lot of [disabled tests](https://github.com/18F/calc/blob/develop/frontend/tests/test_selenium.py#L296-L507) and a few notes about their fragility.

I'm wondering if these tests are even that useful and/or should be fixed at all. Upon first look, I don't think they have much value because the things they look to be testing are already covered by either Jest tests or by the backend python tests.

We might want to decommission that entire file and associated code to run it. Thoughts, @toolness or @hbillings?

**Comment by toolness** [comment]
Sure, given our forsaken troubles with Selenium, SauceLabs, and pretty much everything related to integration testing, I am ok with torpedo-ing this whole doggone file.

**Comment by jseppi** [comment]
🎉🎉🎉

**Comment by jseppi** [comment]
Related #1639 and #330

**Comment by toolness** [comment]
Hmm, so I just realized that we do have a few selenium tests that are actually fairly useful, like running aXe (see #1554).  Granted, those would probably be more reliably run under headless chrome or something (see #1624) since they don't necessarily take much advantage of Selenium itself.

**Comment by jseppi** [comment]
Yeah, maybe we could port those actually useful pieces before removing `test_selenium` and its associates?
